README.md

```diff
@@ -2993,6 +2993,10 @@ Other Style Guides
 
     // good
     this.firstName = 'Panda';
+    
+    // good
```

Comment:
i think this is useful, but we should include a note warning about browser support that links to http://kangax.github.io/compat-table/es6/#test-WeakMap

Reply:
Updated, let me know if that was what you had in mind.